openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
```

Comment:
"iterable thereof" might best be reworded as "Iterable of ResonanceScattering" to match the context throughout the rest of the API